### ISSUE TITLE
Flag

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#742]
- [x] Colour contrast violations:
- Alpha
- Alert
- Warning
- Positive
- Notice
<img width="709" alt="image" src="https://github.com/user-attachments/assets/701bee4d-b881-4d9e-bccf-6c61435298e6" />
